components/SearchBar/SearchBar.tsx

```diff
@@ -96,6 +97,8 @@ const SearchBar = ({
       case "Enter":
         if (query.length > 3) {
           if (event.key === "Enter" && query.length > 3) {
+            tracking.trackEvent({ action: "press-enter", category: "search-bar", label: query });
```

Comment:
I think these actions could be enumerated ```export enum { PRESS_ENTER: "press-enter" ... etc }``` that way we would have a source of truth within the app for our action options, possibly categories as well, and could help passing arbitrary strings in the future (spelling errors etc)

Reply:
thanks, cool, I'll do that!

Reply:
well we should be careful though with naming our events (and possibly categories as well) - although you could argue that using enums sort of forces us to be more careful ...

TBH the parameters for this "tracking API" are heavily influenced by what Google Analytics is using, I would have liked to implement at least one other tracking service in order to validate these choices (i.e. check if this action/category/label convention works equally well with other services) ... but, I think these params are a reasonably obvious choice, can't really go wrong with this:

action - category - label
- or in other words (synonyms) -
event - subject - value

Reply:
Great suggestion, I've just pushed 2 commits to implement this ...